husky_components/tabs/main.js

```diff
@@ -50,6 +50,7 @@ define(function() {
             forceReload: false,
             callback: null,
             forceSelect: true,
+            conditionData: {},
```

Comment:
I don't really think so, sounds more like the condition themselves are passed. Usually it is about forms, that's why I came up with this. What about `passedData` or `testData` (because the described conditions are tested against this data).